pipeline/experiment.py

```diff
@@ -562,6 +565,18 @@ def extract_nidq_trial_data(session_key, channel):
                                        & {**session_key, 'trial_note_type': 'bitcode'}).fetch(
         'trial', 'trial_note', order_by='trial')
 
+    if ephys_bitcodes is None:
+        # no bitcodes (the nidq.XA.txt file for bitcode is not available)
+        if (Session & session_key).fetch1('rig') == 'RRig-MTL' and len(trial_start_times) == len(behav_trials):
+            # for MTL rig, this is glitch in the recording system
+            # if the number of trial matches with behavior, then this is a 1-to-1 mapping
+            log.info('.... Unable to read bitcode! Recognize RRig-MTL session with matching number of behavior and ephys trials, using one-to-one trial mapping')
```

Comment:
might want to log at higher priority e.g. warning

Reply:
I think `info` is fine here, as missing bitcode and using one-to-one mapping is part of the routine, to be expected. The real danger is when there is no bitcode file and the trial numbers don't match (not a one-to-one), for this, error is raised